lib/network/http2_network_config.go

```diff
@@ -31,7 +31,6 @@ func NewHTTP2NetworkConfigFromEndpoint(endpoint *sebakcommon.Endpoint) (config H
 	var ReadTimeout time.Duration = 0
 	var ReadHeaderTimeout time.Duration = 0
 	var WriteTimeout time.Duration = 0
-	var IdleTimeout time.Duration = 5
```

Comment:
@Geod24 For easy source code recognition, IMHO it'd better leave previous code. 
what do you think? this is just suggestion. :)

Reply:
I don't see why ? This is just a variable in the config parsing function.
I left `IddleTimeout` in the structure (L86) even though it is fixed to `0` for the reason you mention, but I don't think it's more readable to have `IdleTimeout: IdleTimeout` and look up `IdleTimeout` just to find out that it's constant.